src/engine/renderer/tr_local.h

```diff
@@ -295,6 +295,8 @@ static inline void glFboSetExt()
 enum class lightMode_t { FULLBRIGHT, VERTEX, GRID, MAP };
 enum class deluxeMode_t { NONE, GRID, MAP };
 
+enum class dynamicLightRenderer_t { FORWARD, TILED };
```

Comment:
This should be renamed from FORWARD to something else to not create confusion with the forward/deferred terminology.

Reply:
I can use “LEGACY”, what would you recommend that describes best the implementation?

Reply:
I think LEGACY is fine, there's no widely-accepted name for it.